packages/telemetry/src/telemetry.ts

```diff
@@ -1,4 +1,5 @@
 import { spawn } from 'child_process'
+import os from 'node:os'
```

Comment:
I'd be careful these - I'm not sure why... but I've found there are some cases where it throws errors (on Windows... if I remember correctly) - and other places the same imports didn't.